VotingApplication/VotingApplication.Web.Api/Scripts/Create.js

```diff
@@ -112,30 +102,11 @@
             }
         };
 
-        var setupTooltips = function() {
-            $(document).tooltip
-
-            function showOrHideElement(show, element) {
-                element.next(".tip").toggle(show);
-            }
-
-            var tooltipTargets = $(".help-message");
-
-            for (var i = 0; i < tooltipTargets.length; i++)
-            {
-                var $hoverTarget = $(tooltipTargets[i]);
-
-                var hideElement = showOrHideElement.bind(null, false, $hoverTarget);
-                var showElement = showOrHideElement.bind(null, true, $hoverTarget);
-                ko.utils.registerEventHandler($hoverTarget, "mouseover", showElement);
-                ko.utils.registerEventHandler($hoverTarget, "mouseout", hideElement);
-                hideElement();
-            }
-        }
-
         self.initialise = function () {
-            self.populateTemplates();
-            setupTooltips();
+            var defaultExpiryDate = moment().add(30, 'minutes');
```

Comment:
This should be done on creation rather than submission to remove the surprise when its a 30 minute timelimit.

It looks like this is handled by the data-binding already though, so i'll remove it here